react/miniproject/components/userdashboard/userScorePaper.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Paper from 'material-ui/Paper';
+
+const style = {
+  height: 150,
+  width: '15%',
+  margin: 25,
+  padding: 0,
+  textAlign: 'center',
+  display: 'inline-block',
+  background: '#b2ebf2',
+};
+
+class UserScorePaper extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div>
+        <Paper style={style} zDepth={3} circle={true}>
```

Comment:
Instead of using multiple `paper component` . Create a single one in another file and import and use it at places you want by passing data. It will increase reusability .